game-app/game-headed/build.gradle

```diff
@@ -5,6 +5,14 @@ plugins {
     id "de.undercouch.download" version "5.6.0"
 }
 
+//Activate the locked apple API with flag '--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED'
```

Comment:
nitpick: this comment might not be the most helpful.

A stronger comment could be something like: "this flag fixes touch gestures in Java-21 (and maybe have a link to some sort of fix documentation explaining the fix)"

Suggestions:
- reword the comment to describe why, specifically that we are fixing touch gestures (maybe also add a link to any helpful pages that describe more info on this parameter)
- avoid repeating the parameter value in the comment (can just be removed, a very quick comment of a few words is ok!)
- recommend to move the comment to be closer to the parameter (ie: move the comment down a few lines)